SUMMARY.md

```diff
@@ -69,10 +69,13 @@
   * [Migration to AEPPlaces](foundation-extensions/places/migration.md)
   * [Places API Reference](foundation-extensions/places/places-usage-reference.md)
   * [ACPPlaces \(iOS\) Reference](foundation-extensions/places/adobe-places.md)
-* [Profile](foundation-extensions/profile/README.md)
-  * [Migrating to AEPUserProfile](foundation-extensions/profile/acpuserprofile-aepuserprofile.md)
+* [Adobe Experience Platform Profile](foundation-extensions/profile/README.md)
```

Comment:
I am afraid this will be confused with AEP User Profile, can we keep this Profile?